src/app/plugins/platform/Docker/plugins/cms/Drupal8/index.ts

```diff
@@ -287,9 +299,21 @@ class Drupal8 extends Generator {
     // Create the service directory if it doesn't exist.
     // If the services directory doesn't exist, Docker fails since it can't mount
     // it as a volume mount.
-    if (!this.existsDestination('services/drupal')) {
-      this.debug('Creating Drupal service directory.');
-      await mkdir(this.destinationPath('services/drupal'), { recursive: true });
+    if (!this.existsDestination('services')) {
+      this.debug(
+        format.info('Creating services directory at %s.'),
+        this.destinationPath('services'),
+      );
+      try {
+        await mkdir(this.destinationPath('services'));
+      } catch (err) {
+        this.log(
+          format.error('Failed to create services directory at %s.'),
+          this.destinationPath('services'),
+        );
+        this.spawnCommandSync('ls', ['-al']);
```

Comment:
A good point. This was a helpful debugging bit I had just left in there, but for posterity some more context would be helpful.

Reply:
I've added a bit of commentary about it inline as well as a to-do note to add more context in the output. Otherwise, I've also wrapped it in a conditional to only show if the debug flag is enabled to further limit confusion from the output without context.